docs/docs-content/registries-and-packs/deploy-pack.md

```diff
@@ -250,17 +269,15 @@ Go ahead and review each of the following five files in the pack.
 
   <br />
 
-  You can optionally define *presets*, which are predefined values to use in the **values.yaml**. You define presets in a separate **presets.yaml** file. The presets become available when you create the cluster profile. Presets facilitate configuring the profile and avoid errors that can happen by manually editing the **values.yaml** file. Refer [Pack Presets](pack-constraints.md#pack-presets) for details and examples of how to define presets. 
+  Optionally, you can define *presets*, which are predefined values to use in the **values.yaml** file. You define presets in a separate **presets.yaml** file. The presets become available when you create the cluster profile. Presets facilitate configuring the profile and avoid errors that can happen by manually editing the **values.yaml** file. Refer to [Pack Presets](pack-constraints.md#pack-presets) for details and examples of how to define presets. 
```

Comment:
Presets are parameters you can expose in the the UI (see screenshot). When you select a preset, you alter the YAML. Take a peek at this **preset.yaml** from the prometheus-operator pack. You'll notice there is are a few parameters available `displayName`, `group`, `remove` and `add`. The last two are important because that's how you specify what YAML to inject or remove. 

You don't have to explain the behavior here, that's something we need to do on a reference page, but you can explain the UI behavior and how it's used to inject UI options (radio buttons) for pack consumers. 

https://github.com/spectrocloud/pax/blob/31ed5670679214c289caeb04fdd74414007e3f35/stable/loggging_and_monitoring/prometheus-operator-46.4.0/presets.yaml#L1-L28


![CleanShot 2023-11-22 at 10 54 45](https://github.com/spectrocloud/librarium/assets/29551334/e0e5a4e0-08be-4135-b807-4783509e5dbe)

Reply:
I rewrote this paragraph after our meeting.